src/odemis/util/test.py

```diff
@@ -43,15 +44,36 @@ def setlimits():
 
 def start_backend(config):
     """
-    Start the backend
+    config (str): path to the microscope config file.
+    Start the backend by checking the currently running backend. Basically 2 cases/scenarios:
+        1. If no running backend => run the requested one.
+        2. If there is running backend =>
+            a. If the running backend is same as requested one => do nothing
+            b. If the running backend is different from the requested one => stop the running, and run the requested one.
+    """
+    # a backend running
+    if driver.get_backend_status() in (driver.BACKEND_RUNNING, driver.BACKEND_STARTING):
+        current_model = model.getMicroscope().model
+        req_model = modelgen.Instantiator(open(config)).ast
+        if current_model == req_model:
+            return
+        else:
+            stop_backend()
```

Comment:
Please add a log to indicate that the backend is already running, but with a different model, so it will be stopped and restarted.